### PR TITLE
APP-183: Fix profiling demo

### PR DESCRIPTION
**Pull request by Dubland**
/cc @Slash32 @RostakaGmfun

### CODE REVIEW

**Comment by RostakaGmfun** [comment]
Retest this please

File: profilingdemo/source/c/src/kaa_demo.c [hunk 1]
```diff
@@ -31,7 +31,7 @@ static kaa_client_t *kaa_client = NULL;
 
 void kaa_demo_print_configuration_message(const kaa_root_configuration_t *configuration)
 {
-    demo_printf("%s - %s - %s\n", configuration->audio_subscription_active ? "true" : "false",
+    printf("%s - %s - %s\n", configuration->audio_subscription_active ? "true" : "false",
                                   configuration->vibro_subscription_active ? "true" : "false",
```
Comment (ohamov) [inline profilingdemo/source/c/src/kaa_demo.c]: indentation

File: scripts/cmakegen/config.yaml [hunk 2]
```diff
@@ -58,7 +58,7 @@ verifiersdemo:
 profilingdemo:
     appname: 'ProfilingDemo'
     features: ['configuration']
-    languages: ['c', 'cpp']
+    languages: ['cpp']
```
Comment (ohamov) [inline scripts/cmakegen/config.yaml]: cpp?

Comment (Dubland) [inline scripts/cmakegen/config.yaml]: for C Profiling Demo created a special build.sh script

**Comment by kaa-leeroy** [comment]
Can one of the admins verify this patch?

**Comment by Dubland** [comment]
Fixed

**Review by Slash32 — APPROVED** [review]

**Pull request merged**